metricbeat/scripts/docs_collector.py

```diff
@@ -151,21 +166,22 @@ def collect(beat_name):
             if release != "ga":
                 metricset_file += "{}[]\n\n".format(get_release(metricset_fields))
 
-            modules_list[module]["metricsets"][metricset]["release"] = release
+            modules_list[module['name']]["metricsets"][metricset]["release"] = release
```

Comment:
`modules_list[module['name']]` referenced lots of times, it could be created as a single object and assigned to the dict when finished.